### ISSUE TITLE
False positive for UnusedMethod

### ISSUE THREAD

**Issue by aipng** [slevomat/coding-standard#172]
Having this code

```php
class Foo
{

  public static createFromArray($array)
  {
    $new = new self;
    $new->privateMethod();

    return $new;
  }

  private function privateMethod()
  {
    //...
  }
}
```

I believe I get false positive from SlevomatCodingStandard.Classes.UnusedPrivateElements.UnusedMethod when running CodeSniffer.

slevomat/coding-standard: 3.0.4

**Comment by aipng** [comment]
Thanks, works perfectly.
